ts/textsecure/WebSocket.ts

```diff
@@ -54,10 +54,16 @@ export function connect<Resource extends IResource>({
     ...extraHeaders,
     'User-Agent': getUserAgent(version),
   };
+
+  const http_agent_options = {
```

Comment:
I am not a TypeScript ninja, so let me know if there is a better way to do this. My desire here is to add the connect_timeout_ms option to the AgentOptions argument without overwriting any of the defaults.